lib/msal-browser/src/index.ts

```diff
@@ -42,5 +43,7 @@ export {
     Logger,
     LogLevel,
     // Protocol Mode
-    ProtocolMode
+    ProtocolMode,
+    // Utils
+    UrlString
```

Comment:
This change will need to be made on the dev branch before putting out the next release of angular. Is there a 2nd PR for this?

Reply:
Yes, good point! I put a PR up now, #2680